### ISSUE TITLE
SKR PRO v1.1 PlatformIO cant upload thru USB

### ISSUE THREAD

**Issue by cabbagecreek** [bigtreetech/BIGTREETECH-SKR-PRO-V1.1#210]
I have a board that have been in the drawer a wile. This weekend i decided to update my MKS Gen 1.4 to SKR PRO v1.1

I did everything from scratch with a fresh Marlin "Marlin 2.0 with bug fixes" download.

I got ebrything building but the Upload button in "Auto build Marlin" plugin will not work.

I did the SD card update from "firmware.bin" and got the FW uploaded and are able to connect with the USB from the PC

What have I done wrong so the SKR PRO v1.1 is not able to be uploaded from "Auto build Marlin" plugin in PlatformIO?

**Comment by radek8** [comment]
Did you solve your problem?
If so, close the ticket

**Comment by cabbagecreek** [comment]
> Did you solve your problem?
> If so, close the ticket

No.. I cant update from PlatformIo thru "Auto build Marlin" and the USB.

**Comment by agai99** [comment]
This board only flash manually with sd card boot flash
